src/LinkDotNet.NCronJob/Execution/JobExecutor.cs

```diff
@@ -80,15 +86,15 @@ private async Task ExecuteJob(JobExecutionContext runContext, IJob job, AsyncSer
 
             stoppingToken.ThrowIfCancellationRequested();
 
-            await AfterJobCompletionTask(null, stoppingToken);
+            await AfterJobCompletionTask(null, serviceScope, stoppingToken);
         }
         catch (Exception exc) when (exc is not OperationCanceledException or AggregateException)
         {
             // This part is only reached if the synchronous part of the job throws an exception
-            await AfterJobCompletionTask(exc, default);
+            await AfterJobCompletionTask(exc, serviceScope, default);
         }
         // This needs to be async otherwise it can deadlock or try to use the disposed scope, maybe it needs to create its own serviceScope
-        async Task AfterJobCompletionTask(Exception? exc, CancellationToken ct)
+        async Task AfterJobCompletionTask(Exception? exc, AsyncServiceScope serviceScope, CancellationToken ct)
```

Comment:
`serviceScope` seems unused

Reply:
Yeah I missed that. The last commit I tried undoing some work on moving the service scope to the job execution context, but then ended up not needing it. I'll clean it up.